src/routes/patreon.ts

```diff
@@ -0,0 +1,81 @@
+import express from 'express';
+import { patreonAgent } from '../clients/patreon/patreonAgent';
+
+const router = express.Router();
+
+router.get('/oauth/redirect', async (req, res) => {
+  console.log('Start: Link to Patreon...');
+  const { code, state } = req.query;
```

Comment:
```suggestion
  const { code: string, state } = req.query;
```
Does this work? If not let's split the two out to separate lines and type it accordingly to remove 
`      code as string,`
on L31